### ISSUE TITLE
Only 2 projects tasks

### ISSUE THREAD

**Issue by Bogdancev** [stuartpittaway/diyBMSv4Code#113]
Hi Stuart
I followed your video and downloaded/opened the project in PlatformIO, but contrary to what is in the video I only see 2 tasks, there is no controller task, nor other module versions tasks.
Is this correct or my setup is wrong or the project has chanced since the video was produced?

![BuildTasks](https://user-images.githubusercontent.com/20501551/121204775-6bca1000-c877-11eb-8318-35cabcb2bb8a.png)
Regards, Sergii

**Comment by stuartpittaway** [comment]
The video is a little out of date now as platformio keeps changing.

You can use the pre-generated files from GitHub, take a look at the instructions there.

Alternatively, edit the platformio.ini file and remove any of the module versions you don't want.

**Comment by Bogdancev** [comment]
Above happened when I opened a workspace as in the video, and after looking closer I saw that "attiny841_V400" was a task from "ATTINYPIDTuning" project. That project is the top one in the list on workspace projects.

Then I used "File -> Open Folder" instead of "File -> Open Workspace" and opened "ATTINYCellModule" folder and after that all tasks of different module verstions became visible.
![ModuleTasks](https://user-images.githubusercontent.com/20501551/121305713-37e5fd80-c8fe-11eb-9a59-94aad0a91da6.png)

So I'm not sure maybe there is a way to specify which project of workspace projects is active at the moment or maybe there is another way to control which project's tasks are shown.

I will use it as "Open Folder", it builds fine.
